### PR TITLE
Formatting and other small stuff

### PR DESCRIPTION
**Pull request by mmattel**
This PR is based on the work of @tdaget at PR #36361(closed)
The referenced PR is now closed because of rebasing issues and CI will never get green there.

## Description

Fix some small formatting issues and minor corrections

## Related Issue

No related issue

## Motivation and Context

Improve code and readability

## How Has This Been Tested?

Run local

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: No changes needed
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by tdaget** [comment]
You can add a typo correction in tests/data/app/invalid-info.xml
line 2: add `>` char at the end of `<info>``

Thanks

**Comment by mmattel** [comment]
> tests/data/app/invalid-info.xml

I think that `<info` is correctly wrong to show that the content is invalid.
But I can´t decide on that in detail - therefor not corrected.

File: settings/templates/help.php [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
 		<li>
 			<a href="https://owncloud.org/support" target="_blank" rel="noreferrer">
-				<?php p($l->t('Online documentation')); ?> ↗
+				<?php p($l->t('Online documentation')); ?>
 			</a>
 		</li>
 		<li>
 			<a href="https://central.owncloud.org" target="_blank" rel="noreferrer">
-				<?php p($l->t('Forum')); ?> ↗
+				<?php p($l->t('Forum')); ?>
```
Comment (phil-davis) [inline settings/templates/help.php]: The  ↗ character has been removed in various places, like this line.
Is there a reason for that?

**Review by phil-davis — COMMENTED** [review]
And I raised issue #36381 because it seems that `settings/templates/help.php` is not accessible any more. So someone needs to decide what the requirement is for it.

File: settings/templates/help.php (hunk 1 above)
Comment (mmattel) [reply settings/templates/help.php]: Yes, `↗` is not part of any parsable html code, most likely a leftover from an editor.
Removing it is cleaning up stuff.

**Comment by mmattel** [comment]
@phil-davis is codecov a necessity in this case and if, how to make that green ?

File: settings/templates/help.php (hunk 1 above)
Comment (phil-davis) [reply settings/templates/help.php]: The  ↗ character does actually display on the UI. I guess it was put there "back in the old days"  to indicate that this is a link and the user will be taken somewhere when they click it.
Probably should be indicated in a different way "these days".
"somebody" will need to decide exactly what should be done.

**Comment by phil-davis** [comment]
> @phil-davis is codecov a necessity in this case and if, how to make that green ?

The files edited here are really `.html` files that have "old fashioned" embedded PHP. But they have `.php` file extension, so `phpunit` includes them in coverage calculations.

I guess somebody will override the codecov result for a PR  like this.

File: settings/templates/help.php (hunk 1 above)
Comment (phil-davis) [reply settings/templates/help.php]: This needs someone with knowledge of project history to say why those "little arrow" characters were chosen to use here. And then to say what should be done nowadays for that sort of thing.

**Comment by DeepDiver1975** [comment]
old aka dead -> close

**Pull request closed without merging**